server/src/controllers/poaching-incident.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { PoachingIncidentService } from '../services/poaching-incident.service';
+import { PoachingIncidentType } from "../entity/poaching-incident-type.entity";
```

Comment:
Single quotes

Reply:
Removed this import, not used in the class